Model/IO/Export/Nastran/BulkData/LongBulkData.cpp

```diff
@@ -0,0 +1,31 @@
+//*************************
+//Created by Dominik Kopaczka
+//*************************
+
+#include "LongBulkData.h"
+
+using namespace MeshIS::Model::Export::Nastran::BData;
+
+LongBulkData::LongBulkData()
```

Comment:
If you are not creating any special constructor/destructor etc. just don't define them as empty. Consider using =default option to avoid typing empty code.